services/api/jobos_api/codex_adapter.py

```
@@ -314,6 +364,24 @@ async def handle_notification(self, method: str, params: dict[str, object]) -> N
+                and params.get("willRetry") is True
+                and isinstance(provider_turn_id, str)
+            ):
+                self._rate_limit_settling.add(provider_turn_id)
+                asyncio.create_task(
+                    self._settle_provider_rate_limit_retry(
+                        self._thread_id,
+                        provider_turn_id,
+                        self._jobos_turn_id,
+                        params,
+                    )
+                )
```

Ruff (0.16.2)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)